cores/esp32/esp32-hal-touch.c

```diff
@@ -148,18 +143,7 @@ static void __touchInit()
     // Touch Sensor Timer initiated
     touch_pad_set_fsm_mode(TOUCH_FSM_MODE_TIMER);  // returns ESP_OK
     touch_pad_fsm_start();                         // returns ESP_OK
-
-    // Initial no Threshold and setup - TOUCH0 is internal denoise channel
-    for (int i = 1; i < SOC_TOUCH_SENSOR_NUM; i++) {
-        __touchInterruptHandlers[i].fn =  NULL;
-        touch_pad_config(i);                       // returns ESP_OK
-    }
-    // keep ISR activated - it can run all together (ISR + touchRead())
```

Comment:
is there any downside to init ISR here? Seems that you only needed to delete `touch_pad_config(i);` to resolve the issue.

Reply:
ah ok.